[WIP] 🧪 Experimental Table (#718)
vtex/styleguide: react/components/Table/README.md

}
;<ResourceListExample />
```

#### Compound Toolbar

> ⚠️ You can choose the compound OR toolbar prop aproach, never the two at the same time.

Reviewer: ```suggestion
⚠️ You have to choose between the compound or the toolbar prop approach.
```

}
;<ResourceListExample />
```

#### Composable Toolbar

> TODO

Author: Yep, I'm doing it right now
